.buildkite/scripts/build_component.ps1

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env powershell
+
+#Requires -Version 5
+
+param (
+    # The name of the component to be built. Defaults to none
+    [string]$Component,
+    # The base hab version to run the build with. Defaults to latest
+    [string]$BaseHabVersion="latest",
+    # The builder channel to pull from. Defaults to stable
+    [string]$SourceChannel="stable"
+)
+
```

Comment:
Better to add `$ErrorActionPreference="stop"` right here and remove the occurences of `-ErrorAction Stop`. You really don't ever want a non-terminating error here.